project/Build.scala

```diff
@@ -116,6 +118,14 @@ object MyScalaJSPlugin extends AutoPlugin {
           "->https://raw.githubusercontent.com/scala-js/scala-js/v" +
           scalaJSVersion + "/"
         })
+      },
+
+      testHtmlJSDom in Test := {
+        import sys.process._
+        val runner = (testHtml in Test).value.data.getAbsolutePath()
+        val code = Seq("node", "scripts/test-html.js", runner).!
```

Comment:
I'm always a bit wary of the `scala.sys.process` API. It's an API that no one really maintains, and that I hope will eventually go away. In this case we can just as easily do it with Java's `ProcessBuilder`:
```scala
val code = new ProcessBuilder("node", "scripts/test-html.js", runner)
  .inheritIO()
  .start()
  .waitFor()
```

WDYT?